lib/src/customer_io.dart

```diff
@@ -60,6 +60,7 @@ class CustomerIOSDK extends ChangeNotifier {
           enableInApp: true,
           region: Region.us,
           //config options go here
```

Comment:
all properties above this comment are also configurable. Is there a reason that the comment is added in the mid?

Reply:
The comment was just added to differentiate the properties that normally do not need modifications

Reply:
Okay! I get that as "normally do not need modifications _in a sample app_" ! Makes sense!